src/database.ts

```diff
@@ -55,24 +65,110 @@ import {
   TransactionRunner,
 } from './transaction-runner';
 
-export interface GetDatabaseOptions {
+import {google as spannerClient} from '../proto/spanner';
+import {Schema, BasicResponse} from './common';
+import {ServiceError, CallOptions} from 'grpc';
+import {Readable, Transform} from 'stream';
+import {Stats} from 'fs';
+import {StreamProxy} from 'google-gax/build/src/streamingCalls/streaming';
```

Comment:
I don't think we should import this way. Maybe we should just import `Duplex` and make a union type to include `cancel()`.

Reply:
Done!!